ios/VoiceActivityDetector.m

```diff
@@ -0,0 +1,28 @@
+#include "webrtc/common_audio/vad/include/webrtc_vad.h"
+#import "VoiceActivityDetector.h"
+
+@implementation VoiceActivityDetector {
+    VadInst *vad;
+}
+
+- (instancetype)init {
+
+    self = [super init];
+    if(self) {
+
+        WebRtcVad_Create(&vad);
+        WebRtcVad_Init(vad);
+        WebRtcVad_set_mode(vad, 0);
+    }
+    return self;
+}
+
+- (void)dealloc {
+    WebRtcVad_Free(vad);
+}
+
+- (int)isVoice:(const int16_t*)audio_frame sample_rate:(int)fs length:(int) frame_length {
```

Comment:
is there a way to prettify the spacing for these files somehow? or is this correct spacing (seems to vary on this line in particular)

Reply:
Yeah, need to look into that. I think xcode probably has some formatting options